### DISCUSSION TITLE
新艾利都公民的义务与权力

### DISCUSSION THREAD

**Discussion by Soldier11-ObsidianBarracks** [share121/inter-knot#1288] [其他]
凡具有新艾利都都籍的人都是新艾利都公民。

新艾利都公民在法律面前一律平等。

新艾利都尊重和保障人权。

任何公民享有宪法和法律规定的权利，同时必须履行宪法和法律规定的义务。

新艾利都年满十八周岁的公民，不分民族、种族、性别、职业、家庭出身、宗教信仰、教育程度、财产状况、居住期限，都有选举权和被选举权；但是依照法律被剥夺政治权利的人除外。

新艾利都公民有言论、出版、集会、结社、游行、示威的自由。

新艾利都公民有宗教信仰自由。

任何新艾利都辖区机关、社会团体和个人不得强制公民信仰宗教或者不信仰宗教，不得歧视信仰宗教的公民和不信仰宗教的公民。

新艾利都保护正常的宗教活动。任何人不得利用宗教进行破坏社会秩序、损害公民身体健康、妨碍国家教育制度的活动。

宗教团体和宗教事务不受外国势力的支配。

新艾利都公民的人身自由不受侵犯。

任何公民，非经新艾利都人民检察院批准或者决定或者新艾利都人民大法院决定，并由公安机关执行，不受逮捕。

禁止非法拘禁和以其他方法非法剥夺或者限制公民的人身自由，禁止非法搜查公民的身体。

新艾利都公民的人格尊严不受侵犯。禁止用任何方法对公民进行侮辱、诽谤和诬告陷害。

新艾利都公民的住宅不受侵犯。禁止非法搜查或者非法侵入公民的住宅。

新艾利都公民的通信自由和通信秘密受法律的保护。除因新艾利都辖区安全或者追查刑事犯罪的需要，由公安机关或者检察机关依照法律规定的程序对通信进行检查外，任何组织或者个人不得以任何理由侵犯公民的通信自由和通信秘密。

新艾利都公民对于任何国家机关和国家工作人员，有提出批评和建议的权利；对于任何国家机关和国家工作人员的违法失职行为，有向有关国家机关提出申诉、控告或者检举的权利，但是不得捏造或者歪曲事实进行诬告陷害。

对于公民的申诉、控告或者检举，有关辖区机关必须查清事实，负责处理。任何人不得压制和打击报复。

由于辖区机关和新艾利都辖区工作人员侵犯公民权利而受到损失的人，有依照法律规定取得赔偿的权利。

新艾利都公民有劳动的权利和义务。

新艾利都通过各种途径，创造劳动就业条件，加强劳动保护，改善劳动条件，并在发展生产的基础上，提高劳动报酬
和福利待遇。

劳动是一切有劳动能力的公民的光荣职责。新艾利都国有企业和城乡集体经济组织的劳动者都应当以新艾利都主人翁的态度对待自己的劳动。新艾利都提倡社会主义劳动竞赛，奖励劳动模范和先进工作者。新艾利都提倡公民从事义务劳动。

新艾利都对就业前的公民进行必要的劳动就业训练。

新艾利都劳动者有休息的权利。

新艾利都发展劳动者休息和休养的设施，规定职工的工作时间和休假制度。

新艾利都依照法律规定实行企业事业组织的职工和国家机关工作人员的退休制度。退休人员的生活受到国家和社会的保障。

新艾利都公民在年老、疾病或者丧失劳动能力的情况下，有从国家和社会获得物质帮助的权利。国家发展为公民享受这些权利所需要的社会保险、社会救济和医疗卫生事业。

新艾利都和社会保障残废军人的生活，抚恤烈士家属，优待军人家属。

新艾利都和社会帮助安排盲、聋、哑和其他有残疾的公民的劳动、生活和教育。

新艾利都公民有受教育的权利和义务。

新艾利都培养青年、少年、儿童在品德、智力、体质等方面全面发展。

新艾利都公民有进行科学研究、文学艺术创作和其他文化活动的自由。新艾利都对于从事教育、科学、技术、文学、艺术和其他文化事业的公民的有益于人民的创造性工作，给以鼓励和帮助。

新艾利都妇女在政治的、经济的、文化的、社会的和家庭的生活等各方面享有同男子平等的权利。

新艾利都保护妇女的权利和利益，实行男女同工同酬，培养和选拔妇女干部。

婚姻、家庭、母亲和儿童受新艾利都的保护。

夫妻双方有实行计划生育的义务。

父母有抚养教育未成年子女的义务，成年子女有赡养扶助父母的义务。

禁止破坏婚姻自由，禁止虐待老人、妇女和儿童。

下奥地利公民在行使自由和权利的时候，不得损害国家的、社会的、集体的利益和其他公民的合法的自由和权利。

新艾利都公民有维护国家统一和全国各民族团结的义务。

新艾利都公民必须遵守宪法和法律，保守国家秘密，爱护公共财产，遵守劳动纪律，遵守公共秩序，尊重社会公德。

新艾利都公民有维护祖国的安全、荣誉和利益的义务，不得有危害祖国的安全、荣誉和利益的行为。

保卫新艾利都、抵抗侵略是新艾利都每一个公民的神圣职责。

依照法律服兵役和参加民兵组织是新艾利都公民的光荣义务。

新艾利都公民有依照法律纳税的义务。
（我草朋友们，我才发现我之前写的字全打错了）

**Comment by lllliufeng** [comment]
第一句换漏了兄弟

**Reply by Soldier11-ObsidianBarracks** [reply]
感谢提醒

**Comment by Jiucheng258** [comment]
哼，还好我不是新艾丽都人，怎么多条条框框的麻烦死了。

**Reply by Soldier11-ObsidianBarracks** [reply]
请问您登记入境了吗？如果没有登记我们将会视为偷渡，请前往新艾丽都治安局任意一个分局进行登记。

**Comment by Soldier11-ObsidianBarracks** [comment]
税务局真要查查远景实业了，不仅老板逃跑，恐怕税务也有问题
